deploy/packages/bbp-packages.yaml

```diff
@@ -120,7 +120,7 @@ packages:
       - nest@2.18.0
       - py-efel@3.0.80
       - py-bglibpy@4.2.23 ^neuron%intel@19.0.4
-      - py-bluepy@0.14.6
+      - py-bluepy@0.14.7
```

Comment:
8?

Reply:
Yes I added it in the future commit but I did not pushed yet due to pb with synapsetools